transmogrifier/config.py

```diff
@@ -83,10 +85,30 @@ def get_transformer(source: str) -> type:
     return getattr(source_module, class_name)
 
 
-def load_external_config(path: str) -> dict:
+def load_external_config(file_path: str, file_type: str) -> Union[dict, Tag]:
     """
-    Return dict from JSON config file.
+    Return dict from a config file.
 
     """
-    with open(path, "rb") as json_file:
-        return json.load(json_file)
+    if file_type == "json":
+        with open(file_path, "rb") as json_file:
+            return json.load(json_file)
+    elif file_type == "xml":
+        with open(file_path, "rb") as xml_file:
+            return BeautifulSoup(xml_file, "xml")
+    else:
+        return {}
+
+
+def create_dict_from_xml_config(
+    xml_content: Tag, element_name: str, key_tag: str, value_tag: str
+) -> dict:
+    """
+    Return dict from XML object that is formatted like the country and language code
+    XML files hosted by Library of Congress.
+
+    """
+    config_dict = {}
+    for element in xml_content.find_all(element_name):
```

Comment:
Do we care about the obsolete codes that will not be found here because they're nested further down in the documents? Do we think obsolete language or country codes might exist in any of our records?

Reply:
Good catch, I updated the function